modules/charon-utils/src/main/java/org/wso2/charon3/utils/usermanager/InMemoryUserManager.java

```diff
@@ -131,6 +131,17 @@ public User updateUser(User user, Map<String, Boolean> map)
        }
     }
 
+    public User updateUser(User user, Map<String, Boolean> requiredAttributes,
+                           List<String> allSimpleMultiValuedAttributes)
+            throws CharonException, BadRequestException, NotFoundException {
+
+        if (user.getId() == null) {
+            throw new NotFoundException("No user found with the id : " + user.getId());
```

Comment:
The value here is null right? If so why we try to print the value? I think it should say Empty userID. Also, can we use StringUtils?

Reply:
addressed in https://github.com/wso2/charon/pull/286/commits/7def504e293f80abeda86855908072c643671d50